warehouse/email/__init__.py

```diff
@@ -201,6 +201,18 @@ def send_added_as_collaborator_email(request, user, *, submitter, project_name,
     return {"project": project_name, "submitter": submitter.username, "role": role}
 
 
+@_email("2fa-added")
```

Comment:
I feel like `2fa` should be spelled out as `two_factor` here and other places in this changeset. This is more consistent with what we in other places in the code base.